packages/blockchain-extension/extension/commands/packageSmartContractCommand.ts

```diff
@@ -216,15 +220,77 @@ async function packageJsonNameAndVersion(workspaceDir: vscode.WorkspaceFolder, o
         const message: string = 'Please enter a package name and/or package version into your package.json';
         throw new Error(message);
     }
+
+    const dependencies: any = workspacePackageObj.dependencies;
+    contractAPI = dependencies['fabric-contract-api'];
+
+    if (!contractAPI) {
+        // check if low level chaincode
+        contractAPI = dependencies['fabric-shim'];
```

Comment:
We should check that contractAPI is set (it will either be set with `fabric-contract-api` version of `fabric-shim` version) before we attempt to coerce it and get the version (as this will fail ungracefully otherwise).